components/features/Dashboard/Window/Window.tsx

```diff
@@ -114,6 +114,7 @@ const Window: React.FC<Props> = ({
       if (bounds?.width <= 1024 && !mobileMode) {
         setMobileMode(true);
         fullscreen();
+        rndRef.current?.updatePosition({ x: 0, y: 0 })
```

Comment:
Nice one shad. Thanks for fixing the bugs.

Reply:
^_____^